stock_prices/stock_prices.py

```diff
@@ -1,15 +1,21 @@
-#!/usr/bin/python
+# Stock Prices
+
+prices_arr = [1050, 270, 1540, 2000, 2, 3, 1]
 
-import argparse
 
```

Comment:
I like the readability. It seems to not pass only the first test. When given [10, 7, 5, 8, 11, 9] as the input, the test expects 6 but instead, it receives 10.